src/chrome/content/rules/Blizzard.com.xml

```diff
@@ -0,0 +1,125 @@
+<!--
+		Cert mismatch:
+			- attdist.blizzard.com
+			- legion.campaign.blizzard.com
+			- overwatch.campaign.blizzard.com
+			- dist.blizzard.com
+			- edgecast.blizzard.com
+			- click.em.blizzard.com
+			- view.em.blizzard.com
+			- gamescom.blizzard.com
+			- gc.blizzard.com
+			- llnw.blizzard.com
+			- media.blizzard.com
+			- eu.media.blizzard.com
+			- kr.media.blizzard.com
+			- sea.media.blizzard.com
+			- tw.media.blizzard.com
+			- us.media.blizzard.com
+			- mobile.blizzard.com
+			- shop.blizzard.com
+			- smtp.blizzard.com
+			- ur.blizzard.com
+
+		Connection refused:
+			- ftp-eu.blizzard.com
+			- level3.blizzard.com
+
+		Redirect loop:
+			- eu.blizzard.com
+			- kr.blizzard.com
+			- sea.blizzard.com
+			- tw.blizzard.com
+			- us.blizzard.com
+			- www.blizzard.com
+
+		Timeout:
+			- ads.blizzard.com
+			- cn.kbase.blizzard.com
+			- cn.version.blizzard.com
+			- connect-cn.blizzard.com
+			- connect-ie.blizzard.com
+			- connect-kr.blizzard.com
+			- download.blizzard.com
+			- em.blizzard.com
+			- mta.em.blizzard.com
+			- mta2.em.blizzard.com
+			- mta3.em.blizzard.com
+			- mta4.em.blizzard.com
+			- mta5.em.blizzard.com
+			- mta6.em.blizzard.com
+			- mta7.em.blizzard.com
+			- mta8.em.blizzard.com
+			- mta9.em.blizzard.com
+			- email.blizzard.com
+			- email2.blizzard.com
+			- eu-smtp01.blizzard.com
+			- eu.installers.blizzard.com
+			- eu.version.blizzard.com
+			- eumx01.blizzard.com
+			- eumx02.blizzard.com
+			- eumx03.blizzard.com
+			- eumx04.blizzard.com
+			- exchange.blizzard.com
+			- exchange2.blizzard.com
+			- ftp.blizzard.com
+			- gss1.eu.blizzard.com
+			- gss1.kr.blizzard.com
+			- gss1.us.blizzard.com
+			- gss2.us.blizzard.com
+			- iir.blizzard.com
+			- images.blizzard.com
+			- jira.blizzard.com
+			- mx1.blizzard.com
+			- mx10.blizzard.com
+			- mx11.blizzard.com
+			- mx20.blizzard.com
+			- mx21.blizzard.com
+			- mx30.blizzard.com
+			- mx31.blizzard.com
+			- mx4.blizzard.com
+			- kr.installers.blizzard.com
+			- kr.version.blizzard.com
+			- krmx01.blizzard.com
+			- m.eu.mobileservice.blizzard.com
+			- mobile-service.blizzard.com
+			- mobileconnect.blizzard.com
+			- namx01.blizzard.com
+			- nawebex01.blizzard.com
+			- tw.version.blizzard.com
+			- us.installers.blizzard.com
+			- us.mobile-service.blizzard.com
+			- us.paygate.blizzard.com
+			- us.version.blizzard.com
+			- vpn.connect.blizzard.com
+			- vcse.blizzard.com
+			- vcse03.blizzard.com
+			- vcse04.blizzard.com
+			- vcse05.blizzard.com
+			- vcse06.blizzard.com
+
+-->
+<ruleset name="Blizzard.com">
```

Comment:
What I wanted to say is: could you add the `(partial)` string to the ruleset name?